### PR TITLE
[RW-5654][risk=no] Fix paginator item count on workspace admin page

### PR DESCRIPTION
**Pull request by calbach**
- Actual fix: rows=100
- Related cleanups:
  - page size options are not specified, remove from template
  - switch to attribute syntax for boolean props

### CODE REVIEW

File: ui/src/app/pages/admin/admin-workspace.tsx [hunk 1]
```diff
@@ -195,13 +195,13 @@ const FileDetailsTable = (props: FileDetailsProps) => {
 
   return <FlexColumn>
     <DataTable
+        paginator scrollable
```
Comment (jmthibault79) [inline ui/src/app/pages/admin/admin-workspace.tsx:198]: does our one-prop-per-line standard also apply to these?

**Review by jmthibault79 — APPROVED** [review]
works! I'll search for other DataTables which are broken in this way.

File: ui/src/app/pages/admin/admin-workspace.tsx (hunk 1 above)
Comment (calbach) [reply ui/src/app/pages/admin/admin-workspace.tsx]: Sure, probably we should have a lint rule for this

Comment (jmthibault79) [reply ui/src/app/pages/admin/admin-workspace.tsx]: https://github.com/all-of-us/workbench/pull/6061

**Pull request merged**